### DISCUSSION TITLE
create Azure functions with 4.x runtime

### DISCUSSION THREAD

**Discussion by htalluri9** [cloud-custodian/cloud-custodian#9220] [Azure]
currently azure functions are created with runtime 3.x which are deprecated since Dec 2022.
is there a way to create azure function with 4.x runtime ? 

I have gone thru the documentation, but couldn't find an option to provide runtime in the policy definition.

**Comment by kapilt** [comment]
this will require some code updates

**Comment by amon3k** [comment]
Any plans to update python & Azure function runtime to version 4.x?
